### PR TITLE
Add compatibility with pgss 1.8

### PR DESCRIPTION
**Pull request by rjuju**
See also powa-archivist - https://github.com/powa-team/powa-archivist/pull/36

### CODE REVIEW

File: powa/server.py [hunk 1]
```diff
@@ -141,13 +165,21 @@ class GlobalDatabasesMetricGroup(MetricGroupDef):
...
     total_blks_hit = MetricDef(label="Total hit", type="sizerate",
                                desc="Amount of data found in shared buffers")
     total_blks_read = MetricDef(label="Total read", type="sizerate",
                                 desc="Amount of data found in OS cache or"
                                      " read from disk")
+    wal_records = MetricDef(label="#Wal records", type="integer",
+                            desc="Number of WAL records generated")
+    wal_fpi = MetricDef(label="#Wal FPI", type="integer",
+                        desc="Number of WAL full-page images generated")
+    wal_bytes = MetricDef(label="#Wal FPI", type="size",
```
Comment (anayrat) [inline powa/server.py:181]: Should be `Should be `Wal bytes` ?

Comment (rjuju) [reply powa/server.py:181]: Oops, indeed. fixed!

**Comment by banlex73** [comment]
Hi guys, just noticed that powa-web 4.1.0 still not available through pip
install...

чт, 10 груд. 2020 о 03:48 Julien Rouhaud <notifications@github.com> пише:

**Comment by rjuju** [comment]
Hi @banlex73 

Indeed, sorry about that.  I just published powa-web 4.1.0 and powa-collector 1.1.0 on pypi!

**Comment by banlex73** [comment]
much appreciated!

чт, 10 груд. 2020 о 18:03 Julien Rouhaud <notifications@github.com> пише:

**Comment by banlex73** [comment]
Hi Julien
thank you for adding powa-web 4.1.0 and powa-collector 1.1.0 to pypi!
PS: also noticed that the latest *pg_stat_kcache *is not available via yum


yum list pg_stat_kcache*
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.mirrors.hoobly.com
 * centos-sclo-rh: mirror.mi.incx.net
 * epel: mirror.team-cymru.com
 * extras: mirror.datto.com
 * updates: mirrors.usinternet.com
Installed Packages
pg_stat_kcache12.x86_64
                                                            2.1.3-1.rhel7

                                   @pgdg12
Available Packages
pg_stat_kcache10.x86_64
                                                            2.1.3-1.rhel7

                                   pgdg10
pg_stat_kcache11.x86_64
                                                            2.1.3-1.rhel7

                                   pgdg11
pg_stat_kcache95.x86_64
                                                            2.1.3-1.rhel7

                                   pgdg95
pg_stat_kcache96.x86_64
                                                            2.1.3-1.rhel7

                                   pgdg96

чт, 10 груд. 2020 о 18:03 Julien Rouhaud <notifications@github.com> пише:

**Comment by banlex73** [comment]
Hi Julien
please disregard my message about yum and pg_stat_kcache_12- all good now,
I was able to install it
sudo yum install pg_stat_kcache_12.x86_64

чт, 10 груд. 2020 о 18:03 Julien Rouhaud <notifications@github.com> пише:

**Pull request merged**